### ISSUE TITLE
Иконка приложения на нижней панели Windows

### ISSUE THREAD

**Issue by Bess3** [dvzaerko/CRPT#1]
На Windows 10 иконка приложения не отображается на нижней панели. 
![image](https://user-images.githubusercontent.com/102356513/203961430-303a7490-9997-4f3d-90ac-b8f7bcaef357.png)

**Comment by dvzaerko** [comment]
Да, я заметил баг исправлю в ближайшее время. Изменения будут на гит как
обычно

пт, 25 нояб. 2022 г., 12:21 Maksim Bessonov ***@***.***>:
